File: examples/modulith-clinical-tool-jpa/src/test/java/io/zenwave360/example/clinicaltool/modules/clinical/infrastructure/jpa/inmemory/DoctorRepositoryInMemory.java
package io.zenwave360.example.clinicaltool.modules.clinical.infrastructure.jpa.inmemory;

import io.zenwave360.example.clinicaltool.modules.clinical.core.domain.*;
import io.zenwave360.example.clinicaltool.modules.clinical.core.outbound.jpa.DoctorRepository;
import java.math.*;
import java.time.*;
import java.util.*;

public class DoctorRepositoryInMemory extends InMemoryJpaRepository<Doctor> implements DoctorRepository {}

File: examples/modulith-klinical-tool-jpa/src/test/kotlin/io/zenwave360/example/clinicaltool/modules/clinical/infrastructure/jpa/inmemory/DoctorRepositoryInMemory.kt
package io.zenwave360.example.clinicaltool.modules.clinical.infrastructure.jpa.inmemory

import io.zenwave360.example.clinicaltool.modules.clinical.core.domain.*
import io.zenwave360.example.clinicaltool.modules.clinical.core.outbound.jpa.DoctorRepository
import java.math.*
import java.time.*
import java.util.*

class DoctorRepositoryInMemory : InMemoryJpaRepository<Doctor, Long>(), DoctorRepository {
}
